core/platform_feature_list.py

```diff
@@ -21,7 +21,7 @@
 from core.domain import platform_parameter_list as params
 
 
-PARAM_NAMES = params.PARAM_NAMES # pylint: disable=invalid-name
+ParamNames = params.ParamNames # pylint: disable=invalid-name
```

Comment:
Is this name still invalid? (cf the pylint pragma)

Reply:
@Rostkowski Please don't resolve comments. It's important to follow the instructions [here](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve), otherwise your PR review will encounter delays.

Also, just a note, I don't actually know whether the pylint pragma is needed here or not. Make sure to run the lint checks on your local machine to find out, if you haven't done so already! (Instructions for this should be in the wiki and also at the top of scripts/pre_commit_linter.py.)